packages/aws-cdk/bin/cdk.ts

```diff
@@ -62,7 +62,7 @@ async function parseCommandLineArguments() {
     .option('path-metadata', { type: 'boolean', desc: 'Include "aws:cdk:path" CloudFormation metadata for each resource (enabled by default)', default: true })
     .option('asset-metadata', { type: 'boolean', desc: 'Include "aws:asset:*" CloudFormation metadata for resources that uses assets (enabled by default)', default: true })
     .option('role-arn', { type: 'string', alias: 'r', desc: 'ARN of Role to use when invoking CloudFormation', default: undefined, requiresArg: true })
-    .option('toolkit-stack-name', { type: 'string', desc: 'The name of the CDK toolkit stack', requiresArg: true })
+    .option('toolkit-stack-name', { type: 'string', desc: 'The name of the CDK toolkit stack (only applies to \'cdk bootstrap\' and \'cdk deploy\' with legacy synthesis)', requiresArg: true })
```

Comment:
Wouldn't it make sense to just put this under the specific subcommands and not as a global switch?